src/scalars/mod.rs

```diff
@@ -0,0 +1,33 @@
+use std::collections::HashMap;
+use std::sync::Arc;
+
+use async_graphql_value::ConstValue;
+use lazy_static::lazy_static;
+
+pub use crate::scalars::email::Email;
+
+mod email;
+
+lazy_static! {
+    pub static ref CUSTOM_SCALARS: HashMap<String, Arc<dyn Scalar + Send + Sync>> = {
```

Comment:
```suggestion
    static ref CUSTOM_SCALARS: HashMap<String, Arc<dyn Scalar + Send + Sync>> = {
```

Reply:
I am using this in `is_scalar` function

```rust
pub fn is_scalar(type_name: &str) -> bool {
    ["String", "Int", "Float", "Boolean", "ID", "JSON"].contains(&type_name)
        || CUSTOM_SCALARS.keys().any(|v| (*v).eq(type_name))
}
```
it would be tedious to maintain several lists of scalars